### DISCUSSION TITLE
[ FF7 ] Game crashes on launch

### DISCUSSION THREAD

**Discussion by Cesfajardo** [julianxhokaxhiu/FFNx#529] [Q&A] [answered]
**PLEASE NOTE:** If you come here to report a crash while you were using 7th Heaven, please do on their relative support channels. NO SUPPORT will be provided, and your issue will be close immediately.

**Describe the bug**
It does not load, the save, the game initializes normally after starting a new game, but the save loads them and throws a crash error, it has happened twice, once on Steam Deck, where I lost the game, and then on PC , where in the same way the game is crashed at a certain point. The first time, it happened before reactor part 5 and on PC near district 5. In both I did normal save, but now it doesn't work anymore :(

**To Reproduce**

1. Go to 'start game'
2. Click on 'save.'
3. Click on '1st save.'

**Expected behavior**
What should happen is that it works normally, but I've already tried all the settings in the launcher but nothing, it's getting to that point and the save crashes

**Screenshots**
![image](https://user-images.githubusercontent.com/123332758/213941695-edf30790-6ac2-49cb-b205-e2d64f6d7e65.png)
![image](https://user-images.githubusercontent.com/123332758/213941716-cc2a5b41-d442-4ac9-83c6-bca812f06f16.png)

**GPU (please complete the following information):**

- RTX 3080ti
- 528.02

**Additional context**
Please Help, I don't wanna start again from zero :(

**Comment by julianxhokaxhiu** [comment, marked as answer]
Please consider reporting bugs to the relative support channels of the mod manager you're using. No direct FFNx support is provided unless you run through the provided installation methods.
